common-arcana.lic

```diff
@@ -3,7 +3,7 @@
   Documentation: https://elanthipedia.play.net/Lich_script_development#common-arcana
 =end
 
-custom_require.call(%w[common events spellmonitor drinfomon])
+custom_require.call(%w[common common-items events spellmonitor drinfomon])
```

Comment:
@KatoakDR - Does this create the infinite loop with arcana -> items -> travel -> arcana?

Reply:
I've not had any startup issues, but yes, it does create a circular dependency. Hrmmm... I can try to test without it, I'll need to stop all my auto starts b/c I imagine common-items is being started by something else for me already and not being an issue at this point.

Reply:
Well... good news and bad news.

Good news, I was able to reproduce the circular dependency issue when `common-items` is included in the require list. I removed all autostart scripts from my character's config then reconnected then ran `common-arcana` and Lich broke. 🔥 

Bad news, when I remove `common-items` from the require list and repeat my test steps then when the code gets to the point where it needs to use DRCI then it fails because that file hasn't been loaded into memory yet.
```
> ,e echo DRCA.charge?('cambrinth manacle', 1)

--- Lich: exec1 active.

[exec1]>charge my cambrinth manacle 1

Try though you may, you find it too clumsy to charge the cambrinth manacle while wearing it.
> 
| *** Your arcana skill is too low to charge your cambrinth while worn

--- Exception: uninitialized constant DRCA::DRCI
Did you mean?  DRC
               DRCA

common-arcana:327:in `charge?'

--- Lich: exec1 has exited.
```
Line 327 mentioned above is `unless DRCI.in_hands?(cambrinth)`

Reply:
However, there are other parts in common-arcana that use `DRCI` before I suggested added `common-items` to the require list, so I suspect folks must be running some other script(s) that are loading that file before they get to this point.

And since the Lich community isn't complaining about those other DRCI method calls, then this _might_ be an acceptable risk. As in, what are the odds that you _never_ use Lich in a way that loads `common-items` before you get the `you find it too clumsy` error message charging worn cambrinth?

Reply:
I think we can submit this then, and I'll put some time into fixing the circular dependency next. Should be easy, since most of the fixes can be as simple as moving around some things in various commons.